### PR TITLE
refactor: use app.setup instead of manually creating flows

### PR DESCRIPTION
**Pull request by jupyterjazz**
Closes https://github.com/jina-ai/now/issues/1022

---

- [x] This PR references an open issue
- [x] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -182,11 +182,6 @@ def get_aws_info():
     )
 
 
-@pytest.fixture
-def random_index_name():
```
Comment (makram93) [inline tests/conftest.py:186]: Any reason why we moved this away from being a fixture?

Comment (jupyterjazz) [reply tests/conftest.py:186]: yeah, there's no need for that, reverted it back

**Review by makram93 — APPROVED** [review]
LGTM!
Will solve failing e2e local test in a separate PR. 
#1073

**Review by violenil — APPROVED** [review]
I like it! Great work🚀

**Pull request merged**